test_scripts/Defects/8_0/3788/003_revoke_RC_permission_for_App_after_PTU_with_MEDIA_appHMIType.lua

```diff
@@ -0,0 +1,47 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/3788
+---------------------------------------------------------------------------------------------------
+-- Description: Check SDL revoke RC permissions for the App after PTU with 'MEDIA' appHMIType
+-- in case App is registered with 'REMOTE_CONTROL' appHMIType
```

Comment:
I would propose to rewrite description in a following way:
> Check SDL revokes RC permissions for the App in case:
> - Initially app has registered with 'REMOTE_CONTROL' appHMIType
> - appHMIType is changed to 'MEDIA' within PTU